task-definition.migration.json

```diff
@@ -232,7 +232,7 @@
                 }
             ],
             "entryPoint": ["sh", "-c"],
-            "command": ["./bin/docker-migrate && python manage.py migrate_clickhouse"],
+            "command": ["./bin/migrate"],
```

Comment:
thinking of deploying this now. final sanity check - how does this look @guidoiaquinti @tiina303 @fuziontech